### PR TITLE
Re-add cgroupsv2 periodic

### PR DESCRIPTION
**Pull request by fgimenez**
Adds cgroupsv2 periodic now that the fix for the recurring error has been merged here https://github.com/kubevirt/kubevirt/pull/6335

/cc @vasiliy-ul @dhiller 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml [hunk 1]
```diff
@@ -1460,3 +1460,50 @@ periodics:
...
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+        env:
+          - name: TARGET
+            value: "k8s-1.20"
+          - name: KUBEVIRT_CGROUPV2
+            value: "true"
+          - name: KUBEVIRT_E2E_SKIP
+            value: "Multus|SRIOV|GPU|Macvtap|\\[sig-operator\\]"
```
Comment (vasiliy-ul) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml:1498]: I think makes sense to adjust the skip-list a bit:
```suggestion
            value: Multus|SRIOV|GPU|Macvtap|MediatedDevices|QUARANTINE
```

Comment (fgimenez) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml]: Good catch! We don't need to add `QUARANTINE` here, this is handled separately. I've removed `\\[sig-operator\\]` and added `\\[sig-operator\\]`, PTAL.

**Pull request merged**